Finmer.Editor/FormMain.cs

```diff
@@ -282,6 +282,23 @@ private bool SaveAs()
             if (dlgSave.ShowDialog() != DialogResult.OK)
                 return false;
 
+            else 
+            {
+                // Pull the directory we're saving in, see if there's any other finmer projects already there.
+                string save_path = Path.GetDirectoryName(dlgSave.FileName);
+                string[] file_entries = Directory.GetFiles(save_path, "*.fnproj");
+                if (file_entries.Length >= 1)
+                {
+                    // Alert user that saving here will delete unusued json files in the directory, have them acknowledge that fact before continuing.
+                    DialogResult dr = MessageBox.Show("Multiple .fnproj files detected. Saving this data will delete unused .json files in directory. This will delete data on the older projects. Are you sure you wish to proceed?", "Finmer Editor", MessageBoxButtons.YesNoCancel, MessageBoxIcon.Warning);
```

Comment:
The prompt here is technically correct, however I think the fnproj/json files are an implementation detail that may confuse the average user (_"what's an 'unused .json file'?"_). Instead, I'd rather describe the problem itself. How about something like:

> The selected folder contains another Finmer project. This is not supported by the Editor: files from the other project will be overwritten and/or deleted. Continuing anyway may cause unrecoverable data loss. Are you sure you wish to proceed?

Reply:
Supplied the new prompt, commiting changes.